cmd/osmosisd/cmd/root.go

```diff
@@ -102,8 +102,6 @@ func initAppConfig() (string, interface{}) {
 	// server config.
 	srvCfg := serverconfig.DefaultConfig()
 	srvCfg.API.Enable = true
-	srvCfg.StateSync.SnapshotInterval = 1500
-	srvCfg.StateSync.SnapshotKeepRecent = 2
```

Comment:
I think this is good for snapshot interval, tbh I'd even consider making it longer